core/ir/x86/decode_table.c

```diff
@@ -75,7 +75,7 @@
 const instr_info_t * const op_instr[] =
 {
     /* OP_INVALID */   NULL,
-    /* OP_UNDECODED */ NULL,
+    /* OP_UNDECODED */ (instr_info_t[]){{1, 0xffffff, 0, "<undecoded>", 0, 0, 0, 0, 0, 0, 0, 0}},
```

Comment:
I wonder if some code somewhere takes op_instr[op] as non-NULL to mean something.

Reply:
I wonder the same.
Perhaps we should just add an `if (opcode == OP_UNDECODED) return "<undecoded>"` in `decode_opcode_name()` before we query `op_instr[]` and get NULL?

Reply:
How about putting that special case only if op_instr[] is NULL: then the branch added to every call is at least checking for NULL which seems worthwhile.

Reply:
Yeah, much better that way.
Done.